packages/boot/src/booters/booter-utils.ts

```diff
@@ -54,10 +55,7 @@ export function loadClassesFromFiles(
   for (const file of files) {
     debug('Loading artifact file %j', path.relative(projectRootDir, file));
     const moduleObj = require(file);
-    // WORKAROUND: use `for in` instead of Object.values().
-    // See https://github.com/nodejs/node/issues/20278
-    for (const k in moduleObj) {
-      const exported = moduleObj[k];
+    for (const [k, exported] of Object.entries(moduleObj)) {
```

Comment:
`for-in` loop is fine for this usage. I don't think we need to go back to `Object.entries`.

Reply:
Ok ! So what about removing the comment as it is now resolved in V8 ? This is what confused me in the first place.